### PR TITLE
gelu example && TensorRefGeLu

### PR DESCRIPTION
**Pull request by jiyang1011**

### CODE REVIEW

File: tools/util/include/cutlass/util/reference/device/tensor_relu.h [hunk 1]
```diff
@@ -112,6 +112,62 @@ struct TensorReLuFunc {
   }
 };
 
+template <
```
Comment (aacostadiaz) [inline tools/util/include/cutlass/util/reference/device/tensor_relu.h:115]: Could you move all the gelu related definitions in this file into a tensor_gelu.h file?

**Review by aacostadiaz — COMMENTED** [review]
Look good! Thanks for adding the example. 

I left a small comment.

File: tools/util/include/cutlass/util/reference/device/tensor_relu.h [hunk 2]
```diff
@@ -147,4 +147,4 @@ namespace sycl {
   cutlass::reference::device::detail::TensorReLuFunc<float,
                                                       cutlass::layout::RowMajor>::Params> : std::true_type {};
 }
-#endif
+#endif
```
Comment (aacostadiaz) [inline tools/util/include/cutlass/util/reference/device/tensor_relu.h:150]:
```suggestion
#endif

```

**Review by aacostadiaz — APPROVED** [review]
Looks good. Thanks for addressing the comment

**Review by taozha2 — APPROVED** [review]

**Pull request merged**